docs/kit/logistics/kit-testing.md

```diff
@@ -0,0 +1,221 @@
+# Kit Testing
+
+Prior to SR2025, Kit Collation and Kit Testing were separate events.
+From SR2025, we have combined the two events to reduce the number of times that volunteers need to travel for kit events over the summer.
+The key improvements that have allowed for this are the increased automation of the test scripts and the included functionality of recording board asset codes during testing.
```

Comment:
Nit: I don't think this statement is necessary. There's more to kit collation than just testing. And there's more to inventorise than just the tested boards.

Reply:
Those are also covered on this page. This is explaining why the process has been changed so others can understand.